sdks/opik_optimizer/src/opik_optimizer/algorithms/meta_prompt_optimizer/ops/candidate_ops.py

```diff
@@ -5,27 +5,27 @@
 """
 
 import ast
-from typing import Any
-from collections.abc import Callable
-import logging
 import json
-import random
+import logging
+from collections.abc import Callable
+from typing import Any
+
+from litellm.exceptions import BadRequestError
 
-from ....api_objects import chat_prompt
 from .... import _llm_calls
+from ...._llm_calls import StructuredOutputParsingError
+from ....api_objects import chat_prompt
```

Comment:
If it's more than 2 '..' it's better to just use absolute import `from opik_optimizer.api_objects import chat_prompt`